llarp/nodedb.cpp

```diff
@@ -84,8 +83,7 @@ llarp_nodedb::RemoveIf(std::function<bool(const llarp::RouterContact& rc)> filte
         ++itr;
     }
   }
-
-  disk->addJob(std::bind(&KillRCJobs, files));
+  disk(std::bind(&KillRCJobs, files));
```

Comment:
`KillRCJobs` is a two line function and is only used here, maybe worth just inlining it here:
```c++
disk([files=std::move(files)] {
  for (const auto& file : files)
    fs::remove(file);
});
```